stairsplus/registrations.lua

```diff
@@ -74,3 +74,40 @@ for _, name in pairs(default_nodes) do
 		})
 	end
 end
+
+-- wool registrations
+
+if minetest.get_modpath("wool") then
+
+	local colorlist = {
+		{"white",      "White"},
+		{"grey",       "Grey"},
+		{"black",      "Black"},
+		{"red",        "Red"},
+		{"yellow",     "Yellow"},
+		{"green",      "Green"},
+		{"cyan",       "Cyan"},
+		{"blue",       "Blue"},
+		{"magenta",    "Magenta"},
+		{"orange",     "Orange"},
+		{"violet",     "Violet"},
+		{"brown",      "Brown"},
+		{"pink",       "Pink"},
+		{"dark_grey",  "Dark Grey"},
+		{"dark_green", "Dark Green"},
+	}
+
+	for i in ipairs(colorlist) do
+		local color = colorlist[i][1]
+		local colordesc = colorlist[i][2]
+		
+		stairsplus:register_all("wool", color, "wool:"..color, {
+			description = colordesc.." Wool",
```

Comment:
Can you move the " Wool" part to the strings above? Concatenating strings this way is not I18N-friendly.